requirements/edx/base.txt

```diff
@@ -22,6 +22,8 @@
     # via -r requirements/edx/github.in
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@2d176468e33c0713c911b563f8f65f7cf232f5b6#egg=xblock-google-drive
     # via -r requirements/edx/github.in
+-e git+https://github.com/open-craft/edx-ora2.git@tecoholic/BB-6151-upgrade-tinymce-to-v5#egg=ora2
+    # temporary - for testing
```

Comment:
Leaving a comment to remember that we need to synchronize merging this with https://github.com/openedx/edx-ora2/pull/1869.

Reply:
@Agrendalath I have updated the ORA2 Versions in 82a5cb3a14